website_portal_contract/static/src/js/tour_test_contract.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2017 Laslabs Inc.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_portal_contract.tour_test_contract', function(require) {
+    'use strict';
+
+    var tour = require('web_tour.tour');
+    var base = require('web_editor.base');
+
+    tour.register(
+        'test_contract_view',
+        {
+            url: '/my/home',
+            name: 'Test website portal contract view',
+            test: true,
+            wait_for: base.ready(),
+        },
+        [
+            {
+                content: 'Click on Your Contracts',
+                trigger: "a:contains('Your Contracts')",
+            },
```

Comment:
@lasley so I built this tour for the js testing. This first step passes, however the next step fails I believe because it's a new url page.

I want the trigger to redirect the url to a new page, but not sure if this is the way to do that. Is this correct?